.github/workflows/ci.yaml

```diff
@@ -359,7 +359,12 @@ jobs:
           path: ./.yarn/
           restore-keys: |
             ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
+      - name: Installing AreTheTypesWrong Library
+        run: npm i -g @arethetypeswrong/cli
       - run: ./tools/ci.sh
+      - run: yarn lerna exec 'npm pack'
```

Comment:
@ruzell22 Please run this as part of the script so that the script is self contained. You can run shell commands from NodeJS the same way you already do when invoking attw